.drone.yml

```diff
@@ -0,0 +1,35 @@
+---
+kind: pipeline
+type: docker
+name: default
+
+steps:
+
+  - name: k3d_lint
+    image: golang:1.14
+    commands:
+      - make ci-setup
+      - make check-fmt lint
+
+  - name: k3d_tests
+    image: docker:19.03
+    volumes:
+      - name: dockersock
+        path: /var/run
+    commands:
+      - apk add git bash curl sudo jq make
+      - sleep 5 # give docker enough time to start
+      - make e2e
+
+services:
+  # Starting the docker service to be used by dind
+  - name: docker
+    image: docker:19.03-dind
+    privileged: true
+    volumes:
+      - name: dockersock
+        path: /var/run
+
+volumes:
+  - name: dockersock
```

Comment:
not sure on this one, might need the full path on the host like we do here: https://github.com/rancher/dapper/blob/master/.drone.yml#L105-L108

Reply:
How I understand it is:
- The `docker` service starts with `dind`, creating a docker socket in the `dockersock` tempfs volume
- other containers (steps) mount this volume and thus can make use of the `docker` service's docker socket (shared via the tempfs volume)

Reply:
makes sense